ultravox/data/registry.py

```diff
@@ -0,0 +1,397 @@
+    name="commonvoice-it",
+    base="commonvoice",
+    subset="it",
+    splits=[types.DatasetSplitConfig(name="train", num_samples=169_771)],
+)
+
+CV_JA_CONFIG = types.DatasetConfig(
+    name="commonvoice-ja",
+    base="commonvoice",
+    subset="ja",
+    splits=[types.DatasetSplitConfig(name="train", num_samples=10_039)],
+)
+
+CV_PT_CONFIG = types.DatasetConfig(
+    name="commonvoice-pt",
+    base="commonvoice",
+    subset="pt",
+    splits=[types.DatasetSplitConfig(name="train", num_samples=21_968)],
+)
+
+CV_RU_CONFIG = types.DatasetConfig(
+    name="commonvoice-ru",
+    base="commonvoice",
+    subset="ru",
+    splits=[types.DatasetSplitConfig(name="train", num_samples=26_377)],
+)
+
+GS_XL_CONFIG = types.DatasetConfig(
+    name="gigaspeech",
+    path="speechcolab/gigaspeech",
+    subset="xl",
+    splits=[
+        types.DatasetSplitConfig(name="train", num_samples=1_000_000),
+        types.DatasetSplitConfig(name="validation", num_samples=10_000),
+    ],
+    assistant_template="{{text_proc.format_asr_text(text)}}",
+    transcript_template="{{text_proc.format_asr_text(text)}}",
+)
+
+LS_BASE_CONFIG = types.DatasetConfig(
+    name="librispeech",
+    path="fixie-ai/librispeech_asr",
+    assistant_template="{{text_proc.format_asr_text(text)}}",
+    transcript_template="{{text_proc.format_asr_text(text)}}",
+)
+
+LS_CLEAN_CONFIG = types.DatasetConfig(
+    name="librispeech-clean",
+    base="librispeech",
+    subset="clean",
+    splits=[
+        types.DatasetSplitConfig(name="train.100", num_samples=28_539),
+        types.DatasetSplitConfig(name="train.360", num_samples=104_014),
+    ],
+)
+
+LS_OTHER_CONFIG = types.DatasetConfig(
+    name="librispeech-other",
+    base="librispeech",
+    subset="other",
+    splits=[
+        types.DatasetSplitConfig(name="train.500", num_samples=148_688),
+    ],
+)
+
+PS_CLEAN_CONFIG = types.DatasetConfig(
+    name="peoplespeech",
+    path="fixie-ai/peoples_speech",
+    subset="clean",
+    splits=[
+        types.DatasetSplitConfig(name="train", num_samples=1_000_000),
+        types.DatasetSplitConfig(name="validation", num_samples=10_000),
+    ],
+)
+
+# SODA_CONFIG = types.DatasetConfig(
+#     name="soda",
+#     path="fixie-ai/soda-audio",
+#     splits=[
+#         types.DatasetSplitConfig(name="train", num_samples=1_000_000),
+#         types.DatasetSplitConfig(name="validation", num_samples=10_000),
+#     ],
+#     # Need way to specify message history.
+#     audio_field="audio_second_last_turn",
+#     assistant_template="{{alt_last_turn}}",
+#     transcript_template="{{turns[-2]}}",
+# )
+
+VP_EN_CONFIG = types.DatasetConfig(
+    name="voxpopuli-en",
+    path="facebook/voxpopuli",
+    subset="en",
+    splits=[
+        types.DatasetSplitConfig(name="train", num_samples=1_000_000),
+        types.DatasetSplitConfig(name="validation", num_samples=10_000),
+    ],
+    assistant_template="{{raw_text}}",
+    transcript_template="{{raw_text}}",
+)
```

Comment:
should `voxpopuli-en` be defined in a similar way as `commonvoice-en`, starting from a base config named `voxpopuli`?